### PR TITLE
Add "Add Asset Criticality Tag with InsightVM from Slack"

### PR DESCRIPTION
**Pull request by PhilippBehmer**
WWF

### CODE REVIEW

File: workflows/Add_Asset_Criticality_Tag_in_InsightVM_from_Slack/workflow.spec.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+extension: workflow
+products: ["insightconnect"]
+name: Add_Asset_Criticality_Tag_in_InsightVM_from_Slack
+title: "Add Asset Criticality Tag in InsightVM from Slack"
+description: Set criticality tag from slack with interactive slack message.
```
Comment (jschipp-r7) [inline workflows/Add_Asset_Criticality_Tag_in_InsightVM_from_Slack/workflow.spec.yaml:5]:
```suggestion
description: Set criticality tag from Slack with interactive slack message.
```

**Review by jschipp-r7 — CHANGES_REQUESTED** [review]
Can we change `in` references to `with` to match the other workflows?

<Verb> with <Product> from <Chat Tool>

File: workflows/Add_Asset_Criticality_Tag_in_InsightVM_from_Slack/help.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Description
+
+Set criticality tag from Slack with interactive Slack message.
```
Comment (lmilby-r7) [inline workflows/Add_Asset_Criticality_Tag_in_InsightVM_from_Slack/help.md:3]:
```suggestion
Sets criticality tag from Slack with an interactive Slack message.
```

File: workflows/Add_Asset_Criticality_Tag_in_InsightVM_from_Slack/help.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+* [Slack](https://insightconnect.help.rapid7.com/docs/configure-slack-for-chatops)
+* [Rapid7 InsightVM](https://www.rapid7.com/products/insightvm/)
+
+# Documentation
+
+## Setup
+
+Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
+
+After import, activate the workflow in order to trigger it.
```
Comment (lmilby-r7) [inline workflows/Add_Asset_Criticality_Tag_in_InsightVM_from_Slack/help.md:24]:
```suggestion
After the workflow has been import, activate the it in order to enable the ability to trigger it.
```

File: workflows/Add_Asset_Criticality_Tag_in_InsightVM_from_Slack/help.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+### Usage
+
+*This workflow will trigger in any direct messages to your Chatbot **or** any message in a channel directed @ your Chatbot. Note the Chatbot must be in the channel in order to trigger the workflow this way.*
+
+To run the workflow, send a direct message to your InsightConnect Slack Chatbot or @ your Chatbot in a public channel starting with the command `crit-tag-asset`.
```
Comment (lmilby-r7) [inline workflows/Add_Asset_Criticality_Tag_in_InsightVM_from_Slack/help.md:30]:
```suggestion
To trigger the workflow, send a direct message to your InsightConnect Slack Chatbot or @ your Chatbot in a public channel starting with the command `crit-tag-asset`.
```

**Comment by PhilippBehmer** [comment]
- Applied your changes
- Changed WF name from in to with
- Plugins updated and cloud plugins used
- Switch from extract plugin to Slack "words" output
- Remove filter step, use the timeout path
- New screenshots

@jonschipp @sengleson-r7 please review

File: workflows/Add_Asset_Criticality_Tag_with_InsightVM_from_Slack/workflow.spec.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+hub_tags:
+  use_cases: [vulnerability_management]
+  keywords: [slack, chat, chatops, work_from_home, tags]
+  features: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-workflows/tree/master/workflows/Add_Asset_Criticality_Tag_in_InsightVM_from_Slack
```
Comment (jschipp-r7) [inline workflows/Add_Asset_Criticality_Tag_with_InsightVM_from_Slack/workflow.spec.yaml:15]:
```suggestion
  source_url: https://github.com/rapid7/insightconnect-workflows/tree/master/workflows/Add_Asset_Criticality_Tag_with_InsightVM_from_Slack
```

**Review by jschipp-r7 — APPROVED** [review]

**Pull request merged**